build.gradle

```diff
@@ -36,6 +41,9 @@ plugins {
     id 'java'
 }
 
+ext.cloudType = hasProperty('cloudType') ? cloudType : 'default'
```

Comment:
should we have the default be "local" - I think we talked about moving the cloud / datastore over to its own extension. We already have a local worker too: https://github.com/google/data-transfer-project/tree/master/extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local